cognite/powerops/client/shop/shop_run_api.py

```diff
@@ -37,7 +37,7 @@ def __init__(self, client: CogniteClient, dataset_id: int, cogshop_version: str
     def _get_shopfile_metadata(self, file_external_id: str) -> FileMetadata:
         return self._cdf.files.retrieve(external_id=file_external_id)
 
-    def trigger_case(self, case: Case, shop_version: str) -> list[SHOPRun]:
+    def trigger_case(self, case: Case, shop_version: str) -> tuple[list, list[SHOPRun]]:
```

Comment:
This is a list of what?
```suggestion
    def trigger_case(self, case: Case, shop_version: str) -> tuple[list[???], list[SHOPRun]]:
```